contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
These are still functions in Kernel32, so they need tests by themselves, why did you remove them?